satellite/audit/reporter.go

```diff
@@ -199,6 +202,9 @@ func (reporter *Reporter) recordPendingAudits(ctx context.Context, pendingAudits
 		}
 	}
 	if len(failed) > 0 {
+		for _, v := range failed {
+			reporter.log.Debug("Error recording Audit Pending Nodes ", zap.Stringer("Node ID", v.NodeID), zap.String("Path", v.Path))
```

Comment:
I think in general a good message for these logs would be `failed to record x y z`.

Reply:
updated